ntfysh_client/NotificationDialog.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+using System.ComponentModel;
+using Microsoft.Win32;
+using System.Diagnostics;
+
+
+namespace ntfysh_client
+{
+    public partial class NotificationDialog : Form
+    {
+        [DllImport("user32.dll", CharSet = CharSet.Auto)]
+        private static extern bool AnimateWindow(IntPtr hWnd, int time, int flags);
+
+        private const int ScreenMargin = 20;
+
+        private int _timeout = 0;
+        private System.Timers.Timer? displayTimeoutTimer = null;
+        private System.Windows.Forms.Timer? updateTimer = null;
+        private Stopwatch? shownStopwatch = null;
+        private ToolTipIcon? _icon;
+        private int _progress_value = 0;
+        private int progress
+        {
+            get { return this._progress_value; }
```

Comment:
Could this just read `this.progressBar1.Value` instead?

Reply:
yes, I originally thought there would be more to the setter. Removed.